.vscode/settings.json

```diff
@@ -1,17 +1,27 @@
-{
+{  
+  // This disables vscode from adding lines to files.associations, 
+  // some files might not be recognized though.
+  // This is a vscode issue.
+  "C_Cpp.autoAddFileAssociations": false, 
   "files.associations": {
-    "__config": "cpp",
-    "iosfwd": "cpp",
-    "vector": "cpp",
-    "tuple": "cpp"
-  },
+    "*.c": "c",
+    "*.h": "cpp",
+    "*.cpp": "cpp",
+    "*.hpp": "cpp",
+    "*.cxx": "cpp",
+    "*.hxx": "cpp"
+  }, 
   "editor.formatOnType": true,
   "cmake.preferredGenerators": ["Ninja", "NMake Makefiles"],
-  "cmake.cmakePath": "${workspaceFolder}/.conda/freecad/bin/cmake",
+
+  // Uncomment to use the cmake executable provided by conda
+  //"cmake.cmakePath": "${workspaceFolder}/.conda/freecad/bin/cmake",
```

Comment:
@adrianinsaval @oursland I think there's a misunderstanding, I did not remove conda at all. This is the only place where Ive changed the default to use the system's cmake. This and the "BUILD_WITH_CONDA:BOOL": "ON" just below.

Reply:
I see that you've altered the way in which the python debugger is launched using the added `VSCodeAutostartDebug.FCMacro` file and removed the activation of the conda environment.

The conda devenv will need to be updated to use the `debugpy` package instead of `ptvsd`.  I'll test this change later tonite and provide feedback when I have sufficient time available.

Reply:
sorry the way github displays commented out code confused me, is it not possible to put this stuff into different profiles that can be selected in vscode so that we don't have to manually go an uncomment stuff? I'm not very familiar with vscode but I got interested in trying it out to get a conda dev environment.